src/Libraries/CoreNodeModelsWpf/CustomSelectionViewModel.cs

```diff
@@ -0,0 +1,48 @@
+using CoreNodeModels;
+using CoreNodeModels.Input;
+
+using Dynamo.Core;
+using Dynamo.UI.Commands;
+
+namespace CoreNodeModelsWpf
+{
+    public class CustomSelectionViewModel : NotificationObject
+    {
+        public CustomSelection Model { get; }
+
+        /// <summary>
+        /// This command is bound to the Add button in the GUI
+        /// </summary>
+        public DelegateCommand AddCommand { get; }
+
+        public DelegateCommand RemoveCommand { get; }
+
+        private void AddItem(object obj)
+        {
+            Model.Items.Add(new DynamoDropDownItem(string.Empty, string.Empty));
+        }
+
+        private void RemoveItem(object parameter)
+        {
+            if (parameter is DynamoDropDownItem item)
+            {
+                Model.Items.Remove(item);
+
```

Comment:
funny whitespace - alright, thats my last comment about it.

Reply:
yes space between lines - 
I think you're on the right track - I usually just go by what ms says:
https://docs.microsoft.com/en-us/learn/modules/csharp-readable-code/4-exercise-use-whitespace